charts/jaeger/values.yaml

```diff
@@ -306,6 +318,10 @@ spark:
 # End: Default values for the various components of Jaeger
 
 hotrod:
+  # pod level security context
```

Comment:
Can we move these comments to the README. you can make one line like 
```
<component>.podSecurityContext | Pod security context | `nil`
<component>.securityContext | Container security context | `runAsNonRoot: true`<br>`runAsUser: 1000`
```

Reply:
Sure. Can do that.